### ISSUE TITLE
Question about the CLI.

### ISSUE THREAD

**Issue by Vernetzt** [i2van/hourglass#20]
Going to word this more as a question as this could be me doing something incorrectly but the CLI seems to fail under conditions I assumed would work.

Are these invalid arguments or is this a problem somewhere else? With the examples below I get an "unexpected error" window that crashes before I can click "show details". 

Using current portable release added to PATH. Same results when attempted in the executable's directory as well.

||Command|
|------|------|
| works| hourglass -w minimized 1h40|
| works| hourglass -w minimized -t "test" 1h40|
| fails| hourglass 1h40|
| fails| hourglass -w last 1h40|
| fails| hourglass -t "test" 1h40| 
| fails| hourglass -w normal 1h40|
| fails| hourglass -w normal 5|
| works| hourglass -w minimized 5|
| works| hourglass -w last|
| fails| hourglass -w last 5|

**Comment by i2van** [comment]
I tested all the commands and everything works as expected.

Could you please answer these questions?

1. What is your Windows version? Run `winver` (GUI) or `ver` (CLI) to find out.
2. Which [.NET Framework version](https://learn.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed) is installed? Run `reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full" /s` and look for `Release REG_DWORD`. You might need admin rights for that, so run this command elevated.
3. What is your regional settings?

**Comment by i2van** [comment]
Could you take a look for `Hourglass-Crash*txt` files in `%USERPROFILE%\AppData\Local\Temp`?

**Comment by i2van** [comment]
Looks like you build project yourself. There is a [bug](https://github.com/dotnet/wpf/issues/9582) in VS Preview, [workaround](https://github.com/dotnet/wpf/issues/9582#issuecomment-2295640074).

Released version does not have this issue.

**Comment by Vernetzt** [comment]
Build was grabbed from the current release `HourglassPortable.zip`

Windows 11 Pro 23H2 22631.4317
.NET 4.8.09032
Region US EN
`where` points to the correct location

**Log**
```
System.InvalidOperationException: Cannot perform this operation while dispatcher processing is suspended.
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Threading.DispatcherOperation.Wait(TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.InvokeImpl(DispatcherOperation operation, CancellationToken cancellationToken, TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.Invoke(Action callback, DispatcherPriority priority, CancellationToken cancellationToken, TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.Invoke(Action callback, DispatcherPriority priority)
   at Hourglass.Windows.TimerWindow.OnActivated(EventArgs e)
   at System.Windows.Window.WmActivate(IntPtr wParam)
   at System.Windows.Window.WindowFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.PublicHooksFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   ```

**Comment by i2van** [comment]
Could you please test with binaries from [this build](https://github.com/i2van/hourglass/actions/runs/11460200743)?

Unfortunately, I don't have Win 11 to test.

**Comment by Vernetzt** [comment]
388eb4e seems to fix the issue for me.

Selecting any dark theme option or creating a custom one no longer crashes when using any combination of CLI arguments that previously lead to errors.

**Comment by i2van** [comment]
Fixed in [1.15.41](https://github.com/i2van/hourglass/releases/tag/1.15.41)
